{{cookiecutter.bot_name}}/app/db/models.py

```diff
@@ -0,0 +1,59 @@
+"""Database models declarations."""
+
+from typing import Any
+
+from sqlalchemy import Column, Integer, String, update as sqlalchemy_update
+from sqlalchemy.future import select
+
+from app.db.sqlalchemy import Base, async_db_session
+
+
+class ModelAdminMixin:
+    """Mixin for CRUD operations for models."""
+
+    @classmethod
+    async def create(cls, **kwargs: Any) -> None:
+        """Create object."""
+        async_db_session.add(cls(**kwargs))  # type: ignore
```

Comment:
`update` and `select` is new-style, `add` is oldstyle. Let's use [insert](https://docs.sqlalchemy.org/en/14/core/dml.html#sqlalchemy.sql.expression.insert).